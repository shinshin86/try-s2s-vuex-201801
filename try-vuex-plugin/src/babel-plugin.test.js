import plugin from './babel-plugin'
import pluginTester from 'babel-plugin-tester'

const input = 'import { ADD, DONE, DELETE } from "./mutation-types";'

pluginTester({
  plugin,
  snapshot: true,
  tests: [
    {
      title: 'try-s2s-vuex-201801',
      code: input
    }
  ]
})
