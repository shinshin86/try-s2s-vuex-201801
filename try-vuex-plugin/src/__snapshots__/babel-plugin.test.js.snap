// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` try-s2s-vuex-201801 try-s2s-vuex-201801: try-s2s-vuex-201801 1`] = `
"
import { ADD, DONE, DELETE } from \\"./mutation-types\\";

      ↓ ↓ ↓ ↓ ↓ ↓

import { ADD, DONE, DELETE } from \\"./mutation-types\\";
const state = {};
const actions = {
  [ADD]({
    commit
  }, state) {
    commit(ADD, state);
  },

  [DONE]({
    commit
  }, state) {
    commit(DONE, state);
  },

  [DELETE]({
    commit
  }, state) {
    commit(DELETE, state);
  }

};
const mutations = {
  [ADD](state) {
    state = state;
  },

  [DONE](state) {
    state = state;
  },

  [DELETE](state) {
    state = state;
  }

};
const getters = {};
export default new Vuex.Store({
  state,
  getters,
  actions,
  mutations
});
"
`;
